satellite/console/service.go

```diff
@@ -968,7 +968,8 @@ func (s *Service) isProjectMember(ctx context.Context, userID uuid.UUID, project
 		return result, errs.New(internalErrMsg)
 	}
 
-	for _, membership := range memberships {
+	for _, projectMembership := range memberships {
+		membership := projectMembership
```

Comment:
This is a false positive because after the assignation of the loop declared var the loop ends due to the `return`